### DISCUSSION TITLE
What should we be hidding in?

### DISCUSSION THREAD

**Discussion by inactivelaylabmanley** [Boxnet-Media/boxnetjam#36] [General]
How exactly should the hiding system work both visually and from a gameplay perspective?

**Comment by MichaPika** [comment]
### **Where**

- lockers
- tables
- cabinets

Those were my main thoughts when thinking where to hide, ofcourse can be expanded/changed, those were just first thoughts.

### **How**

when you near an object you can hide in/under, you get the option to in a little popup like "use (character) to hide", also maybe when you first discover the place to hide, it flashes for a second to show you, that you can use it for something (suggested by my partner)

### **Gameplay**

I was thinking maybe when your hiding depending on the object you get a radius that you can see around it, or you can only see a certain direction like if your hiding in a locker you can only see straight out but if your under a table then you can see around. but if your under a table since a table is open all around there's a chance (if you were running from an enemy) that they could find you under it and the chance that they can find you depends on the type of enemy.
